astropy/utils/tests/test_introspection.py

```diff
@@ -65,15 +65,15 @@ def test_find_mod_objs():
 
 
 def test_minversion():
-    from types import ModuleType
-    test_module = ModuleType("test_module")
-    test_module.__version__ = '0.12.2'
-    good_versions = ['0.12', '0.12.1', '0.12.0.dev', '0.12dev']
-    bad_versions = ['1', '1.2rc1']
+    import numpy
```

Comment:
I could not use a fake module anymore with this change, but I did test edge case manually. If there is a way to pass in fake module again, please let me know. Thanks!

With this patch:

```python
>>> from astropy.utils import minversion
>>> import numpy
>>> numpy.__version__
'1.20.2'
>>> minversion(numpy, '1.20.2')
True
>>> minversion(numpy, '1.20.2', inclusive=False)
False
```